cadc-access-control-server/src/main/java/ca/nrc/cadc/auth/AuthenticatorImpl.java

```diff
@@ -147,6 +149,18 @@ public void augmentSubject(final Subject subject)
             PluginFactory pluginFactory = new PluginFactory();
             UserPersistence userPersistence = pluginFactory.createUserPersistence();
             Principal ldapPrincipal = getLdapPrincipal(subject);
+
+            // Remove the HttpPrincipal in subject, if any
```

Comment:
Could you please add a comment as to why the http principal is being removed here?